src/js/control-bar/progress-control/seek-bar.js

```diff
@@ -53,7 +53,6 @@ class SeekBar extends Slider {
   setEventHandlers_() {
     this.update = Fn.throttle(Fn.bind(this, this.update), UPDATE_REFRESH_INTERVAL);
 
-    this.on(this.player_, 'timeupdate', this.update);
```

Comment:
can we keep this line and get rid of line 84 then? 😄

Reply:
Done 👍